packages/ui/node/reporter.ts

```diff
@@ -60,7 +64,16 @@ export default class HTMLReporter implements Reporter {
   async onFinished() {
     const result: HTMLReportData = {
       paths: this.ctx.state.getPaths(),
-      files: this.ctx.state.getFiles(),
+      files: await Promise.all(this.ctx.state.getFiles().map(async (file) => {
```

Comment:
Let's just store a separate field with test files content, and return the content from `readTestFile` fake RPC

Reply:
done